frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -181,7 +181,16 @@ const appendPipelineRunStatus = (pipeline, pipelineRun) => {
     return mTask;
   });
 };
-
+export const hasInlineTaskSpec = (pipeline: K8sResourceKind) => {
+  const tasks = pipeline?.spec.tasks ?? [];
+  let hasInlineSpec = false;
+  tasks.forEach((task) => {
+    if (!hasInlineSpec) {
+      hasInlineSpec = !!(task.taskSpec && !task.taskRef);
+    }
+  });
```

Comment:
You're re-implementing `Array.prototype.some`... best just [use it instead](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some).

Reply:
Updated it.